ground/src/main/java/com/google/android/ground/ui/datacollection/tasks/date/DateTaskFragment.kt

```diff
@@ -54,7 +54,7 @@ class DateTaskFragment : AbstractTaskFragment<DateTaskViewModel>() {
           c[Calendar.DAY_OF_MONTH] = updatedDayOfMonth
           c[Calendar.MONTH] = updatedMonth
           c[Calendar.YEAR] = updatedYear
-          viewModel.updateResponse(c.time)
+          viewModel.updateResponse(getDateFormatter(), c.time)
```

Comment:
Would this work if the value is provided to the task as a default value (e.g. when resuming task from killed state with a draft)? Will the value be formatted correctly?

Reply:
sounds like this is a partial loca we are talking, is it?
because coming from killed state in this pr start everything from start

Reply:
The question I'm trying to ask is for draft load, would the date/time value be displayed correctly when we navigate to that task?